data/cell.hh

```diff
@@ -53,35 +53,33 @@ struct cell {
 
 
     struct tags {
-        class cell;
-        class atomic_cell;
-        class collection;
-
-        class flags;
-        class live;
-        class expiring;
-        class counter_update;
-        class external_data;
-
-        class ttl;
-        class expiry;
-        class empty;
-        class timestamp;
-        class value;
-        class dead;
-        class counter_update;
-        class fixed_value;
-        class variable_value;
-        class value_size;
-        class value_data;
-        class pointer;
-        class data;
-        class external_data;
-
-        class chunk_back_pointer;
-        class chunk_next;
-        class chunk_data;
-        class last_chunk_size;
+        class cell {};
+        class atomic_cell {};
+        class collection {};
+
+        class flags {};
+        class live {};
+        class expiring {};
+        class counter_update {};
+        class external_data {};
+
+        class ttl {};
+        class expiry {};
+        class empty {};
+        class timestamp {};
+        class value {};
+        class dead {};
+        class fixed_value {};
+        class variable_value {};
+        class value_size {};
+        class value_data {};
+        class pointer {};
+        class data {};
+
+        class chunk_back_pointer {};
+        class chunk_next {};
+        class chunk_data {};
+        class last_chunk_size {};
```

Comment:
This could have been done as a separate step but this is a hackaton so I won't bikeshed. :grin:

Reply:
You are right, it could but thanks for not making me to separate to another commit :slightly_smiling_face: